app/components/Header/index.js

```diff
@@ -28,6 +28,36 @@ function Header() {
   );
 }
 
+function Header() {
+  return(
+    <div className="pos-f-t">
+    <div className="collapse" id="navbarToggleExternalContent">
+      <div className="bg-inverse text-white">
```

Comment:
Give these classname `nav-item` and remove the `<br>`s, should display inline with that